web-console/developer/pipelines/pipelines.md

```diff
@@ -1,5 +1,314 @@
 # Pipelines
-The Pipelines page is a primary nav item in the OpenShift Developer perspective.
+The Pipelines page is a primary navigation item in the OpenShift Developer perspective.  Pipelines is only shown when the operator is installed.
```

Comment:
Which operator? Is it called "Pipelines"?

Reply:
right now it's called the "OpenShift Pipelines Operator".  Think i should be a bit more specific but still generic and say "the pipelines operator" ?